### ISSUE TITLE
UI issue: favourite symbol almost indistinguishable whether 'faved' or not

### ISSUE THREAD

**Issue by TwizzyDizzy** [andstatus/andstatus#450]
Hi folks,

I've just started using andstatus and am pretty content so far :) However, one thing has been quite a nuisance: The symbol (heart) for 'faving' a post only shows a slight difference in color, when one 'faves' a post compared to the post not being 'faved'. This is even worse, if the light conditions are bad or brightness of the screen is low. Often I'm running into a situation where I think "did the app take my press of the icon?".

The same problem applies to the "retweet" symbol, though it's not as easy to solve as for the heart, as there is no closed area within the symbol one could fill.

To illustrate, I will attach two pictures. One of status quo, one with a proposed solution:

Before:
![before](https://cloud.githubusercontent.com/assets/179595/24291419/2c799a4c-1089-11e7-9d1a-6ed84232f6d1.png)

Possible solution:
![suggest-after](https://cloud.githubusercontent.com/assets/179595/24291426/328cc85a-1089-11e7-8765-7d818660c14e.png)

One other, rather quick solution might be to just increase the contrast in the 'active' version, maybe with a sharp color like red or so.

Cheers
Thomas

**Comment by yvolk** [comment]
@TwizzyDizzy I agree about the "heart"  symbol, will add "filled heart" easily as it is available in the Google's icon pack.
Regarding `"reblogged/retweeted"` symbol: it is duplicated above a tweet as seen on your screenshot. Anyway, if you will find good alternative icon for the "retweeted" state , please share with me.

**Comment by yvolk** [comment]
Tip: Turn off the "Button bar" below a message ( in Timeline section of Settings), use context menu for favoriting, - and you will get filled heart right now :-)

**Comment by TwizzyDizzy** [comment]
Thanks for the tip, does work for now but as the bottom bar is default (and I do like it), I'd still vote for the change.

As for retweet: I'll have a look whether I can find a better icon.

Concerning the retweet symbol above: both have different functions and the one above has no other state, I guess, while the bottom one would be "active" if I myself had retweeted the post, or am I wrong?

Cheers so far
Thomas

**Comment by yvolk** [comment]
Please note that the Button bar below a message allows you to act as a current User only whereas context menu allows you to "Act as..." alternative User (if you have more than one account in the same Social network) and it has much more options, e.g. 3 Replying options...

Yes, you're right: these buttons are shown "tinted" only if some of your users favorited/retweeted the message.
